templates/static/js/scripts.js

```diff
@@ -177,7 +177,9 @@
 
   // Generate id from H2s content when it does not exist
   document.querySelectorAll('main h2:not([id])').forEach(function (heading) {
-    var id = heading.textContent
+    // Only get direct text from h2 node, excluding any child nodes
```

Comment:
I am guessing you add the status labels to the side navigation dynamically. Would it not be possible to apply a similar logic to the headings within the documentation?

Reply:
Apologies for not being clearer in my comment.

Yes, just wanted to get the thought out there. Definitely out of scope.